publication/src/main/scala/hmda/publication/reports/util/RaceUtil.scala

```diff
@@ -49,7 +49,7 @@ object RaceUtil {
 
       case Joint =>
         larSource.filter { lar =>
-          (applicantTwoOrMoreMinorities(lar) || coApplicantTwoOrMoreMinorities(lar)) &&
+          (applicantOneOrMoreMinorities(lar) || coApplicantOneOrMoreMinorities(lar)) &&
```

Comment:
I'm not sure if this follows the wiki exactly... the `applicantOneOrMoreMinorities` should only check for the race being between 1 and 4 (and is that in any race field, not just `race1`?).

Reply:
For `race2` through `race5`, the only options are 1, 2, 3, 4, and 5. So for those fields, checking that they equal 1,2,3 or 4 is the same as checking `!= 5`